src/main/webapp/app/shared/button/CurationModeButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Button, ButtonProps } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IRootStore } from 'app/stores/createStore';
+import { connect } from '../util/typed-inject';
+import { Link } from 'react-router-dom';
+
+interface CurationModeButtonProps {
+  curateEntityId: string | number;
+}
+
+const CurationModeButton: React.FunctionComponent<
+  CurationModeButtonProps & StoreProps & ButtonProps & React.HTMLAttributes<HTMLButtonElement>
+> = props => {
+  const { toggleCurationPanel, curateEntityId, ...buttonProps } = props;
+  return (
+    <Button tag={Link} to={`/fda-submission/${curateEntityId}/curate`} color="secondary" type="button" {...buttonProps}>
```

Comment:
`/fda-submission/` should be passed through property. We could create entity enum and create route utils to return proper URL based on entity and action. Action should be enum as well.